substrate/client/api/src/backend.rs

```diff
@@ -206,6 +207,11 @@ pub trait BlockImportOperation<Block: BlockT> {
 		state_version: StateVersion,
 	) -> sp_blockchain::Result<Block::Hash>;
 
+	/// Commit complete partial state.
+	/// `sc-client-db` expects blocks with state to be marked.
+	/// Otherwise it complains that state is not found.
```

Comment:
What is a complete partial state? Sounds like an oxymoron deserving a better description.